### DISCUSSION TITLE
v0.10.4

### DISCUSSION THREAD

**Discussion by Blinue** [Blinue/Magpie#763] [Announcements]
**Full Changelog**: https://github.com/Blinue/Magpie/compare/v0.10.3...v0.10.4

<hr /><em>This discussion was created from the release <a href='https://github.com/Blinue/Magpie/releases/tag/v0.10.4'>v0.10.4</a>.</em>

**Comment by hooke007** [comment]
这个更新日志机器人自动生成的吗，好像就是简单直接提取了commit信息，好混乱的感觉

**Reply by Blinue** [reply]
确实是机器人生成的，稍后会改

**Comment by luojunyuan** [comment]
So appreciate for your work.
看到magpie越来越好由衷的感到高兴！感谢作者 Blinue 的各种努力。适配多显示器，多类型的应用，windows的各种情形真的需要花费很多工夫。magpie的 UI 也越来越完美易用。现在magpie是我电脑必备的一款程序，即便不使用我也会下载他。

**Reply by Blinue** [reply]
谢谢！所有贡献、反馈、测试或推广的人都值得感谢。我开发 Magpie 原本是自己使用的，没想到受到了很多关注，这给了我持续开发的动力。未来 Magpie 也将一直开源、免费且纯净。

**Comment by logamrs2** [comment]
Oi pessoal! Tudo bem?
Tem ocorrido uma coisa estranha com o meu App, quando eu ativo o recurso ele duplica a tela, ou seja, fica a versão estendida e a menor por cima! 

Mais alguém passando por isso?

Gostaria de saber se existe alguma forma de limpar o cache do app, pois eu não encontrei o lugar, acredito que isso possa ajudar, Valeu obrigado!

**Reply by Blinue** [reply]
Do you mean the source window is still visible when scaling? This might be because the source window is topmost.

**Comment by Bengala511** [comment]
I have a problem because when i try to scale the game Team Fortress 2 it doesnt scale i customizate the preferences and everything, i have the game on widescreen, with very low resolution, but when i try to scale manualy or with the 3 seconds to scale it just dont work.
